src/Exception/NpmCommandFailedException.php

```diff
@@ -15,12 +15,12 @@ final class NpmCommandFailedException extends Exception
      * @param string         $command The executed command.
      * @param Exception|null $cause   The cause, if available.
      */
-    public function __construct($command, Exception $cause = null)
+    public function __construct($command, $output, Exception $cause = null)
```

Comment:
Safer to have` __construct($command, Exception $cause = null, $output = null)` and `throw new NpmCommandFailedException($command, null, $output);`
But I don't see other places where this constructor is called, so should be fine.